src/js/search-bar/search-bar.controller.js

```diff
@@ -1,13 +1,21 @@
 import {AbstractController} from "../mvc/controller";
+import { selectDomElement } from "../utils/dom-selector";
 import {SearchBarEvent} from "./search-bar.constants";
 
 export class SearchBarController extends AbstractController {
     constructor(model, view) {
         super(model, view);
-        this.registerListener(SearchBarEvent.CLICK_AWESOME_BUTTON, (data) => {this.onClickAwesomeButton(data)})
+        this.registerListener(SearchBarEvent.CLICK_SEARCHBAR_OVERLAY, (data) => { this.onClickSearchBarOverlay(data)})
+        this.registerListener(SearchBarEvent.CLICK_SEARCH, (data) => { this.onClickSearchBtn(data)})
     }
 
-    onClickAwesomeButton(data) {
-        console.log(data);
+    onClickSearchBarOverlay(DOMElement) {
+        DOMElement.classList.add('active-search-bar');
```

Comment:
in this case there is no sense to put DOM element as an argument. we could have here just view method that toggle search bar